### PR TITLE
Update CodeOwners

### PR DESCRIPTION
**Pull request by RickWinter**
Add fields for issue triage assignment

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,33 +6,44 @@
...
 # PRLabel: %Attestation
-/sdk/attestation/        @LarryOsterman @gkostal @anilba06 @ahmadmsft @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+/sdk/attestation/        @gkostal @anilba06 @ahmadmsft @LarryOsterman @rickwinter
 
+# AzureSDKOwners: @gearama
+# ServiceLabel:  %KeyVault
 # PRLabel: %KeyVault
 /sdk/keyvault/           @gearama @antkmsft @rickwinter @LarryOsterman
 
+# AzureSDKOwners: @gearama
```
Comment (ahsonkhan) [inline .github/CODEOWNERS:39]: Do you have a doc or some pointer that explain what this comment now does (along with what `ServiceLabel')?

Does this mean an issue is auto-assigned to this person if is has that service label on it?

**Review by ahsonkhan — APPROVED** [review]
Seems reasonable, though not sure I understand the impact of this change on issue triaging.

**Review by antkmsft — APPROVED** [review]

**Review by antkmsft — COMMENTED** [review]
Add `@gearama` for `/sdk/tables`?

File: .github/CODEOWNERS (hunk 1 above)
Comment (RickWinter) [reply .github/CODEOWNERS:39]: The person is the first person to triage.  It doesn't mean they fix, but they are the first to look at it.

**Comment by RickWinter** [comment]
> Add `@gearama` for `/sdk/tables`?

Coming in seperate PR, I'll also be adding the messaging topics

**Pull request merged**